examples/twr_aloha/Makefile

```diff
@@ -0,0 +1,60 @@
+APPLICATION = twr-aloha
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= dwm1001
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# Include uwb-core, uwb-dw1000
+USEPKG += uwb-core
+USEPKG += uwb-dw1000
+
+# Include all ranging algorithms
+USEMODULE += uwb-core_twr_ss
+USEMODULE += uwb-core_twr_ss_ack
+USEMODULE += uwb-core_twr_ss_ext
+USEMODULE += uwb-core_twr_ds
+USEMODULE += uwb-core_twr_ds_ext
+
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+# Set the device role 0x0 for tag 0x4 for an anchor
+DW1000_ROLE ?= 0x00
+CFLAGS += -DDW1000_ROLE_DEFAULT=$(DW1000_ROLE)
+
+# All uwb-core applications need to enable `-fms-extensions`
+CFLAGS += -fms-extensions
+ifneq (,$(filter llvm,$(TOOLCHAIN)))
+  CFLAGS += -Wno-microsoft-anon-tag
+endif
```

Comment:
I though it could be moved to the package, and it is there as well, but I get the warning when application code makes calls of the api's that have these constructs, I did not find another way to deal with this other than also having the flags in the application. Any ideas on other ways to deal with this?

Reply:
Maybe it could be placed in the package Makefile.include.

Reply:
It is already and it was not enough

Reply:
Ok, so let's leave it also at application level then.